GoogleForCreators/web-stories-wp
# Autoset as Background: Show snackbar message only once

## Context


After setting a background image a message is shown: "Full bleed images and videos are automatically set as background. Double click to scale and position at any time". This message is displayed every time a background image is set. It should be shown only once.

## Summary


A property web_stories_background_is_set_dialog_dismissed is added to localStorage after a background message is shown, so the message is not shown anymore.

## User-facing changes


A background message is shown only once.
![image](https://user-images.githubusercontent.com/2496186/158823894-1eb1bbff-ed5c-4388-b95c-7527bf11fdce.png)
![image](https://user-images.githubusercontent.com/2496186/158824061-e8dd49ea-2a6e-4a27-bc15-737193e6d0cf.png)
![image](https://user-images.githubusercontent.com/2496186/158824193-59c73e81-183c-4f13-9fd4-530ebf328f12.png)


## Testing Instructions



This PR can be tested by following these steps:

1. Add an image to the story.
2. Drag the image to cover whole area of the story so the image is set as a background.
3. A message is shown.
4. A property web_stories_background_is_set_dialog_dismissed is added to localStorage.
5. Remove the background, drag a new image to cover the whole area of the Page so the image is set as a background. 
7. The message should not be shown.

## Checklist




---



Fixes #10782
